ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -239,6 +251,18 @@
     reason="Test will not run on Openshift dedicated cluster",
 )
 
+skipif_msprovider = pytest.mark.skipif(
+    config.ENV_DATA["platform"].lower() in MANAGED_SERVICE_PLATFORMS
+    and config.ENV_DATA["cluster_type"].lower() == "provider",
+    reason="Test will not run on Managed service provider cluster",
+)
+
+skipif_msconsumer = pytest.mark.skipif(
```

Comment:
same here, please call the var `skipif_ms_consumer`

Reply:
modified as suggested